clang/test/SemaCXX/constant-expression-p2280r4.cpp

```diff
@@ -1,6 +1,8 @@
 // RUN: %clang_cc1 -std=c++23 -verify=expected,nointerpreter %s
 // RUN: %clang_cc1 -std=c++23 -verify %s -fexperimental-new-constant-interpreter
 
+// FIXME: P2280R4 should be backported. Run this in C++11 and later modes.
```

Comment:
On top of this FIXME, add RUN lines for C++11 mode, then add expected directives for whatever comes out of it. So that if someone accidentally makes it work, it doesn't go unnoticed.

Reply:
We should instead do that in #129646 (and merge that first)